doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,198 +1,99 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 Perception Pipeline Tutorial
-============================
+==================================
+
+Introduction
+------------
+MoveIt serves sensor plugins for utilizing 3D sensor data in planning pipeline. When it's ran the demo of this tutorial, it should be seen the following rviz screen.
 
-MoveIt allows for seamless integration of 3D sensors using `Octomap <http://octomap.github.io/>`_.
-Once properly configured, you should see something like this in rviz:
+.. raw:: html
 
-.. image:: perception_configuration_demo.png
-   :width: 700px
+        <video width="700px" controls="true" autoplay="true" loop="true">
+            <source src="../../../_static/videos/perception_pipeline_demo.webm" type="video/webm">
+            MoveIt Perception Pipeline Demo
+        </video>
 
 Getting Started
 ---------------
-If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
-
-Configuration
--------------
-
-In this section, we will walk through configuring the 3D sensors on your robot with MoveIt. The primary component in MoveIt that deals with 3D perception is the Occupancy Map Updater. The updater uses a plugin architecture to process different types of input. The currently available plugins in MoveIt are:
```

Comment:
This section shouldn't be removed

Reply:
Added to this part . https://github.com/CihatAltiparmak/moveit2_tutorials/blob/bc1504d3d6c13e17c19bf506fe70358bbc83a100/doc/examples/perception_pipeline/perception_pipeline_tutorial.rst?plain=1#L21-L24